result/wrap-regex.js

```diff
@@ -0,0 +1,13 @@
+function a() {
+    return /foo/.test("bar");
```

Comment:
__[development][eslint]__
* Error - Wrap the regexp literal in parens to disambiguate the slash. (wrap-regex)